components/Navigation.jsx

```diff
@@ -102,53 +110,15 @@ const Navigation = () => {
                 </Row>
               </div>
               <Nav className="navbar-nav-click align-items-lg-center" navbar>
-                <NavItem>
-                  <Link href="/about" passHref>
-                    <NextNavLink className={getNavLinkClass("/about")}>
-                      <span className="nav-link-inner--text">About Us</span>
-                    </NextNavLink>
-                  </Link>
-                </NavItem>
-
-                <NavItem>
-                  <Link href="/team" passHref>
-                    <NextNavLink className={getNavLinkClass("/team")}>
-                      <span className="nav-link-inner--text">The Team</span>
-                    </NextNavLink>
-                  </Link>
-                </NavItem>
-
-                <NavItem>
-                  <Link href="/events" passHref>
-                    <NextNavLink className={getNavLinkClass("/events")}>
-                      <span className="nav-link-inner--text">Events</span>
-                    </NextNavLink>
-                  </Link>
-                </NavItem>
-
-                <NavItem>
-                  <Link href="/publications" passHref>
-                    <NextNavLink className={getNavLinkClass("/publications")}>
-                      <span className="nav-link-inner--text">Publications</span>
-                    </NextNavLink>
-                  </Link>
-                </NavItem>
-
-                <NavItem>
-                  <Link href="/charity" passHref>
-                    <NextNavLink className={getNavLinkClass("/charity")}>
-                      <span className="nav-link-inner--text">Charity</span>
-                    </NextNavLink>
-                  </Link>
-                </NavItem>
-
-                <NavItem>
-                  <Link href="/calendar" passHref>
-                    <NextNavLink className={getNavLinkClass("/calendar")}>
-                      <span className="nav-link-inner--text">Calendar</span>
-                    </NextNavLink>
-                  </Link>
-                </NavItem>
+                {navLinks.map(([text, link], index) => (
+                  <NavItem key={`nav-item-${text}-${index}`}>
```

Comment:
Not important, but the URL in the key is a bit excessive. Surely we can do just the text field, as we don’t want two navbar items with the same name anyway.

Reply:
Isn't that the text field in the key, not the URL? navLinks is in the form `[text, link]`, e.g. `["Team", "/team"]`. In that case, the key would be `nav-item-Team-1`. (Just using index isn't enough because the links can be reordered)